### ISSUE TITLE
MQTT (still) missing from docker build - reopen of #1227

### ISSUE THREAD

**Issue by douginoz** [louislam/uptime-kuma#4859]
### 📑 I have found these related issues/pull requests

Re-open of #1227 as it doesn't appear to have been completed

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

Ubuntu 23.10
Using latest Uptime Kuma docker image: louislam/uptime-kuma:latest

Trying to set up Notification:

Notification Type:  Apprise
Apprise URL: mqtt://blah:blahpass@192.168.1.20:1883/homeassistant/uptime_kuma/plex

Status:  Apprise is installed

Pressing Test produces 'Process Exited with Code 1'
Docker log files show:
```
2024-06-16T00:48:11-07:00 [AUTH] INFO: Login by username + password. IP=192.168.1.35
2024-06-16T00:48:11-07:00 [RATE-LIMIT] INFO: remaining requests: 19
2024-06-16T00:48:11-07:00 [AUTH] INFO: Successfully logged in user moa. IP=192.168.1.35
Error: Process exited with code 1
    at ChildProcess.done (/app/node_modules/promisify-child-process/index.cjs:70:19)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:303:5) {
  code: 1,
  signal: null,
  stdout: '2024-06-16 00:48:21,526 - ERROR - mqtt:// is disabled on this system.\n' +
    '2024-06-16 00:48:21,527 - ERROR - There are no service(s) to notify\n',
  stderr: ''

}
```
Issue #1227 was opened but seems to have been closed automatically, probably because the last response was mistaken for a solution but it was a short-term workaround that no longer works.  

At any rate, the docker build needs to have paho-mqtt installed in the build for everything to work.

Note that there was a couple of people that built their own images (@dadcoachengineer and @agustinvinao ) but these don't work for several reasons on my system (ubuntu 23.10) and were supposed to be short-term workarounds until paho-mqtt was installed on the legit docker image.

### 👟 Reproduction steps

Try to set up an MQTT notification.  It will fail if uptime-kuma is run as a docker container.

### 👀 Expected behavior

All Apprise notifications should work including MQTT

### 😓 Actual Behavior

Pressing Test produces 'Process Exited with Code 1'
Docker log files show:
```
2024-06-16T00:48:11-07:00 [AUTH] INFO: Login by username + password. IP=192.168.1.35
2024-06-16T00:48:11-07:00 [RATE-LIMIT] INFO: remaining requests: 19
2024-06-16T00:48:11-07:00 [AUTH] INFO: Successfully logged in user moa. IP=192.168.1.35
Error: Process exited with code 1
    at ChildProcess.done (/app/node_modules/promisify-child-process/index.cjs:70:19)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:303:5) {
  code: 1,
  signal: null,
  stdout: '2024-06-16 00:48:21,526 - ERROR - mqtt:// is disabled on this system.\n' +
    '2024-06-16 00:48:21,527 - ERROR - There are no service(s) to notify\n',
  stderr: ''

}
```

### 🐻 Uptime-Kuma Version

Version: 1.23.13

### 💻 Operating System and Arch

Ubuntu 23.10

### 🌐 Browser

Chrome, Firefox

### 🖥️ Deployment Environment

Docker version 25.0.2, build 29cf629

### 📝 Relevant log output

```shell
==> Performing startup jobs and maintenance tasks
==> Starting application with user 0 group 0
Welcome to Uptime Kuma
Your Node.js version: 18.20.2
2024-06-16T01:21:32-07:00 [SERVER] INFO: Welcome to Uptime Kuma
2024-06-16T01:21:32-07:00 [SERVER] INFO: Node Env: production
2024-06-16T01:21:32-07:00 [SERVER] INFO: Inside Container: true
2024-06-16T01:21:32-07:00 [SERVER] INFO: Importing Node libraries
2024-06-16T01:21:32-07:00 [SERVER] INFO: Importing 3rd-party libraries
2024-06-16T01:21:33-07:00 [SERVER] INFO: Creating express and socket.io instance
2024-06-16T01:21:33-07:00 [SERVER] INFO: Server Type: HTTP
2024-06-16T01:21:33-07:00 [SERVER] INFO: Importing this project modules
2024-06-16T01:21:34-07:00 [NOTIFICATION] INFO: Prepare Notification Providers
2024-06-16T01:21:34-07:00 [SERVER] INFO: Version: 1.23.13
2024-06-16T01:21:34-07:00 [DB] INFO: Data Dir: ./data/
2024-06-16T01:21:34-07:00 [SERVER] INFO: Connecting to the Database
2024-06-16T01:21:34-07:00 [DB] INFO: SQLite config:
[ { journal_mode: 'wal' } ]
[ { cache_size: -12000 } ]
2024-06-16T01:21:34-07:00 [DB] INFO: SQLite Version: 3.41.1
2024-06-16T01:21:34-07:00 [SERVER] INFO: Connected
2024-06-16T01:21:34-07:00 [DB] INFO: Your database version: 10
2024-06-16T01:21:34-07:00 [DB] INFO: Latest database version: 10
2024-06-16T01:21:34-07:00 [DB] INFO: Database patch not needed
2024-06-16T01:21:34-07:00 [DB] INFO: Database Patch 2.0 Process
2024-06-16T01:21:34-07:00 [SERVER] INFO: Load JWT secret from database.
2024-06-16T01:21:34-07:00 [SERVER] INFO: Adding route
2024-06-16T01:21:34-07:00 [SERVER] INFO: Adding socket handler
2024-06-16T01:21:34-07:00 [SERVER] INFO: Init the server
2024-06-16T01:21:34-07:00 [SERVER] INFO: Listening on 3001
2024-06-16T01:21:34-07:00 [SERVICES] INFO: Starting nscd
Start cloudflared
2024-06-16T08:21:34Z INF Starting tunnel tunnelID=5ba866bd-96e1-4979-ace3-dfed4afe56ad
2024-06-16T08:21:34Z INF Version 2024.4.0
2024-06-16T08:21:34Z INF GOOS: linux, GOVersion: go1.21.5, GoArch: amd64
2024-06-16T08:21:34Z INF Settings: map[no-autoupdate:true token:*****]
2024-06-16T08:21:34Z INF cloudflared will not automatically update if installed by a package manager.
2024-06-16T08:21:34Z INF Generated Connector ID: bf02a824-81c2-4da5-bced-10642b43bdb9
2024-06-16T08:21:34Z INF Initial protocol quic
2024-06-16T08:21:34Z INF ICMP proxy will use 172.18.0.6 as source for IPv4
2024-06-16T08:21:34Z INF ICMP proxy will use :: as source for IPv6
2024-06-16T08:21:34Z INF Starting metrics server on 127.0.0.1:45223/metrics
2024-06-16T08:21:34Z WRN Your version 2024.4.0 is outdated. We recommend upgrading it to 2024.6.0
2024-06-16T08:21:34Z INF Registered tunnel connection connIndex=0 connection=c372f587-2ba9-40a9-a44d-152ef8dbd277 event=0 ip=198.41.200.113 location=sea09 protocol=quic
2024-06-16T08:21:34Z INF Registered tunnel connection connIndex=1 connection=8d6593a8-f8ae-4719-9379-0750e4ec57a8 event=0 ip=198.41.192.67 location=yvr01 protocol=quic
2024-06-16T08:21:35Z INF Registered tunnel connection connIndex=2 connection=5d3be8ad-d42e-4150-9924-36bad3a2dc88 event=0 ip=198.41.200.233 location=sea06 protocol=quic
2024-06-16T01:21:36-07:00 [SOCKET] INFO: New polling connection, IP = 192.168.1.35
2024-06-16T01:21:36-07:00 [AUTH] INFO: Login by token. IP=192.168.1.35
2024-06-16T01:21:36-07:00 [AUTH] INFO: Username from JWT: moa
2024-06-16T01:21:36-07:00 [AUTH] INFO: Successfully logged in user moa. IP=192.168.1.35
2024-06-16T01:21:36-07:00 [SOCKET] INFO: New polling connection, IP = 192.168.1.35
2024-06-16T08:21:36Z INF Registered tunnel connection connIndex=3 connection=d18bb885-1ee5-4657-b0c1-ed97b26e912d event=0 ip=198.41.192.7 location=yvr01 protocol=quic
2024-06-16T01:21:36-07:00 [AUTH] INFO: Login by token. IP=192.168.1.35
2024-06-16T01:21:36-07:00 [AUTH] INFO: Username from JWT: moa
2024-06-16T01:21:36-07:00 [AUTH] INFO: Successfully logged in user moa. IP=192.168.1.35
Error: Process exited with code 1
    at ChildProcess.done (/app/node_modules/promisify-child-process/index.cjs:70:19)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at Socket.<anonymous> (node:internal/child_process:450:11)
    at Socket.emit (node:events:517:28)
    at Pipe.<anonymous> (node:net:350:12) {
  code: 1,
  signal: null,
  stdout: '2024-06-16 01:24:29,110 - ERROR - mqtt:// is disabled on this system.\n' +
    '2024-06-16 01:24:29,110 - ERROR - There are no service(s) to notify\n',

  stderr: ''
}
```

**Comment by louislam** [comment]
If paho-mqtt will not make the docker image much bigger, a pull request is welcome.

**Comment by dadcoachengineer** [comment]
@louislam - I can put the pull in today.

I have been sporadically maintaining the image below but it would be more convenience to have it merged.

https://hub.docker.com/repository/docker/jasonshearer/uptime-kuma-mqtt/general

**Comment by CommanderStorm** [comment]
Resolved by #4878
